docs/source/ops/connect/auth0.rst

```diff
@@ -0,0 +1,657 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Setting Up Auth0
+================
+
+In this section, we will walk through a complete setup of an entire Daml
+Connect system using Auth0 as its authentication provider.
+
+Authentication v. Authorization
+-------------------------------
+
+In a complete Daml system, the Daml components only concern themselves with
+*authorization*: requests are accompanied by a (signed) token that *claims* a
+number of rights (such as the right to *act as*  a given party). The Daml
+system will accept these claims at face value provided that the token signature
```

Comment:
Sounds somewhat weird. "The Daml system will accept these claims at _face value_ provided that the token signature checks out".

Reply:
Hmm... Perhaps "The Daml system will not perform any other verification than checking the signature."?

Reply:
I see, that is a idiom, wasn't aware of that :facepalm: In this case we can leave it like this, however the other sentence you suggested sounds better to me because it is more specific :smile: